ls1intum/Artemis
# Fix submission links from the participations component

### Checklist

### Motivation and Context

The link does not work at the moment.

### Description

We had decided to remove the `/submissions` suffix from the exam routes but didn't account for this more hidden case. This is fixed now. Edit: Also includes some consistency improvements from #3058 now. The Submission page wasn't linking to exam routes before.

### Steps for Testing

1. Go to the participations of an exam exercise
2. Open a submission
3. Check that the same still works for non-exam exercises
4. Repeat the same steps with the submissions page of an exercise (exam and non-exam)

### Screenshots

The following links should work for exams now too:
- Participations page:
![grafik](https://user-images.githubusercontent.com/72132281/111778015-59fa5180-88b4-11eb-817d-7fc939c9b9a5.png)
- Submissions page:
![grafik](https://user-images.githubusercontent.com/72132281/111869956-21c14480-8982-11eb-9807-cec59d353934.png)
